ColorPatterns/Scenes/Game/GameController.swift

```diff
@@ -81,18 +85,9 @@ private extension GameController {
         }
     }
 
-    private func setupSwipeDownGesture(direction: UISwipeGestureRecognizer.Direction) {
-        let swipeGesture = UISwipeGestureRecognizer(target: self, action: #selector(didSwipe(_:)))
-        swipeGesture.direction = direction
-        view.addGestureRecognizer(swipeGesture)
-    }
-
-    @objc func didSwipe(_ sender: UISwipeGestureRecognizer) {
-        updateColorBars()
-    }
-
     private func playSound(soundFile: Int) {
-        if defaults.bool(forKey: K.DefaultsKeys.sound) == true {
+        // TODO: Make a helper class that does all that stuff
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/vandermesis/ColorPatterns/blob/c07cd9a3b7a63e61e86332aeb4fa4cabf8b81c17/ColorPatterns/Scenes/Game/GameController.swift#L89"></span>TODOs should be resolved (Make a helper class that does …).<br />
<code>todo</code> <code>GameController.swift:89</code></td>
    </tr>
  </tbody>
</table>